sdk/tables/azure-data-tables/tests/test_table_entity_cosmos_async.py

```diff
@@ -1,2036 +1,281 @@
-# coding: utf-8
 # -------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
-from uuid import UUID
 import pytest
 
-from datetime import datetime, timedelta, timezone
-from dateutil.tz import tzutc, tzoffset
-from math import isnan
-
 from devtools_testutils import AzureRecordedTestCase
 from devtools_testutils.aio import recorded_by_proxy_async
 
-from azure.data.tables import (
-    TableEntity,
-    EntityProperty,
-    EdmType,
-    UpdateMode,
-    generate_table_sas,
-    TableSasPermissions,
-)
-from azure.data.tables.aio import TableServiceClient
-from azure.core import MatchConditions
-from azure.core.credentials import AzureSasCredential
-from azure.core.exceptions import HttpResponseError, ResourceNotFoundError, ResourceExistsError, ResourceModifiedError
+from azure.core.exceptions import ResourceExistsError, HttpResponseError
+from azure.data.tables.aio import TableServiceClient, TableClient
 
 from _shared.asynctestcase import AsyncTableTestCase
-from _shared.testcase import _encode_base64
 from async_preparers import cosmos_decorator_async
 
-# ------------------------------------------------------------------------------
-# TODO: change to `with table_client as client:` to close sessions
-# ------------------------------------------------------------------------------
-
-TEST_GUID = UUID("c8924069-90ee-4726-83b5-45ff92496126")
-
-
-class TestTableEntityCosmosAsync(AzureRecordedTestCase, AsyncTableTestCase):
```

Comment:
I'm not sure why so much test code has been removed here?

Reply:
Thanks for catching it! It was copied from a wrong test file, it should be correct now. PTAL